pages.de/common/docker.md

```diff
@@ -23,6 +27,10 @@
 
 `docker pull {{image}}`
 
+- Ein nicht verwendetest Image löschen:
```

Comment:
Remember that the pages can have up to 8 examples, otherwise we can't merge the PR.